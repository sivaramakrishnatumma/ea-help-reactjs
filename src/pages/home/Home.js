import React from "react";
import './Home.css';
const Home = () => {
  return (
    <div className="page-container">
      <h1 className="center">Welcome to EA HELP</h1>
    </div>
  );
};

export default Home;
