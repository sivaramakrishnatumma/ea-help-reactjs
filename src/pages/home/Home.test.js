import React from "react";
import { shallow } from "enzyme";
import Home from "./Home";

describe("Home page test case", () => {
  it("should test Home Page", () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
