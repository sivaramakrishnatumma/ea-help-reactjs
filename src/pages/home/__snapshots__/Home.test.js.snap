// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page test case should test Home Page 1`] = `
<div
  className="page-container"
>
  <h1
    className="center"
  >
    Welcome to EA HELP
  </h1>
</div>
`;
